.release/security-scan.hcl

```diff
@@ -36,8 +36,7 @@ container {
 	# periodically cleaned up to remove items that are no longer found by the scanner.
 	triage {
 		suppress {
-			# N.b. `vulnerabilites` is the correct spelling for this tool.
-			vulnerabilites = [
+			vulnerabilities = [
```

Comment:
Is this correct? The comment calls out that vulnerabilites is what the tool is expecting.

Reply:
Deniz linked me to the documentation that this was fixed 🚀

Reply:
Ah yes, we had a misspelling in the triage section https://github.com/hashicorp/security-scanner/blob/main/CONFIG.md 😄 Both are supported for now but it should be `vulnerabilities` instead of `vulnerabilites`